### PR TITLE
Spike ups sync sidecar

### PR DESCRIPTION
**Pull request by pb82**
DO NOT MERGE

POC: deploy a sidecar to sync the config between UPS and mobile clients.

### CODE REVIEW

File: roles/provision-unifiedpush-apb/tasks/main.yml [hunk 1]
```diff
@@ -18,5 +18,8 @@
 - name: "Delete OAuth proxy Template File"
   file: path=/tmp/oauth-proxy-sa.yaml state=absent
 
+- name: Give default user admin permissions
+  shell: 'oc policy add-role-to-user admin system:serviceaccount:{{ namespace }}:{{ proxy_serviceaccount_name }} -n {{ namespace }}'
```
Comment (matzew) [inline roles/provision-unifiedpush-apb/tasks/main.yml:22]: ouch

Comment (matzew) [reply roles/provision-unifiedpush-apb/tasks/main.yml:22]: really `admin` ?

File: roles/provision-unifiedpush-apb/tasks/provision-ups.yml [hunk 2]
```diff
@@ -94,31 +100,33 @@
...
-- name: Create {{ namespace }} PushApplication on service host {{ unifiedpush_service.service.spec.cluster_ip }}
+- name: "Create {{ namespace }} PushApplication on service host {{ unifiedpush_service.service.spec.cluster_ip }}"
   uri:
     url: http://{{ unifiedpush_service.service.spec.cluster_ip }}/rest/applications
     method: POST
     body: '{"name" : "{{ namespace }}", "description" :  "Push application for {{ namespace }}"}'
     validate_certs: no
     body_format: json
     status_code: 201
+  retries: 5
```
Comment (matzew) [inline roles/provision-unifiedpush-apb/tasks/provision-ups.yml:119]: did it time out for you, @pb82  ?

File: roles/provision-unifiedpush-apb/tasks/main.yml (hunk 1 above)
Comment (pb82) [reply roles/provision-unifiedpush-apb/tasks/main.yml:22]: I tried `View` and `Edit` but no luck.

Comment (matzew) [reply roles/provision-unifiedpush-apb/tasks/main.yml:22]: ok... for now ... :)

**Review by matzew — APPROVED** [review]
:+1:

File: roles/provision-unifiedpush-apb/tasks/provision-ups.yml (hunk 2 above)
Comment (pb82) [reply roles/provision-unifiedpush-apb/tasks/provision-ups.yml:119]: I've once seen it return a non-201 response and the APB fail with an error. Thought a few retries can't hurt.

**Pull request merged**